june/configs/config_example.yaml

```diff
@@ -9,7 +9,7 @@ activity_to_super_groups:
 
 time:
   total_days: 30
-  initial_day: "2020-03-01" 
+  initial_day: "2020-03-01 9:00" 
```

Comment:
we can now do this? cool!

Reply:
Yep! codecov decreased I think because of supergroup edits but this function now works - you can see it in the camps that we have config files specifying which hours things are open :)